packages/feedback/src/core/sendFeedback.ts

```diff
@@ -48,18 +51,39 @@ export async function sendFeedbackRequest(
       event: baseEvent,
     });
 
-    if (!feedbackEvent) {
-      return resolvedSyncPromise({});
+    if (client.emit) {
+      client.emit('beforeSendFeedback', feedbackEvent, { includeReplay: Boolean(includeReplay) });
     }
 
-    client.emit('beforeSendFeedback', feedbackEvent, { includeReplay: Boolean(includeReplay) });
+    try {
+      const response = await transport.send(
```

Comment:
So instead of this, I think we can/should just use `client.sendEvent(feedbackEvent)`. This will also take care of https://github.com/getsentry/sentry/issues/66214#issuecomment-1991189610, I believe. Or is there anything that is preventing this?

Reply:
replacing this call seems to work fine, but we've got no `response` now, so the error messages are not going to come from this file.